src/write_gcm.c

```diff
@@ -2282,42 +2283,24 @@ static monitored_resource_t *monitored_resource_create_from_fields(
                                               num_labels);
 }
 
-typedef struct label_node_s
-{
-  label_t label;
-  struct label_node_s *next;
-} wg_mr_label_node_t;
-
 typedef struct {
-  int processing_labels;
-  int total_labels;
-  char *last_key;
+  _Bool processing_labels;
+  char *label_name;
   char *type;
-  wg_mr_label_node_t *head;
+  llist_t *label_list;
 } wg_mr_parse_context_t;
 
-static int wg_mr_handle_string(void * arg, const unsigned char * val,
+static int wg_mr_handle_string(void *arg, const unsigned char *val,
     wg_yajl_callback_size_t length) {
   wg_mr_parse_context_t *ctx = (wg_mr_parse_context_t*) arg;
```

Comment:
Nit: There should be a space before the *. i.e.  `(wg_mr_parse_context_t *) arg;`